### PR TITLE
Mvertens/perl o orefactor

### PR DESCRIPTION
**Pull request by mvertens**
addition of new comments

### CODE REVIEW

File: scripts/create_newcase [hunk 1]
```diff
@@ -188,12 +201,11 @@ sub _create_caseroot_tools
 }
 
 
-
 sub _check_input_options
 {
     my ($Case,$opts) = @_;
     
-    # Values in uppercase here have a direct correspondence to Entries in the case XML files
+    # Values in uppercase here have a direct correspondence to Entries in the caseroot XML files
```
Comment (jedwards4b) [inline scripts/create_newcase:208]: I don't understand this change?

Comment (mvertens) [inline scripts/create_newcase:208]: I thought that case was a bit ambiguous. Its actually the files in the
caseroot directory.

Comment (mvertens) [reply scripts/create_newcase:208]: @Jun Wang - NOAA Affiliate <jun.wang@noaa.gov>  @jedwards4b/cime
<reply@reply.github.com>
 @Ufuk Turuncoglu <turuncu@ucar.edu> @Rocky Dunlap <rsdunlapiv@gmail.com> -
a meeting sounds great. The sooner the better. When are you available this
week?

Comment (rocky) [reply scripts/create_newcase:208]: I think you mentioned the wrong "rocky". I am not rdunlap.

**Pull request merged**